tests/console/zypper_info.pm

```diff
@@ -35,20 +35,24 @@ sub test_srcpackage_output {
     check_srcpackage_output 'coreutils', $info_output_coreutils;
 
     if (is_sle '>=12-SP3') {
-        if (is_sle '<15') {
-            register_product;
+        if (get_var('FLAVOR') !~ /-Updates$|-Incidents$/) {
```

Comment:
Notice: This will work as long a we don't get any
```bash
FLAVOR="Something-Updates-Incidents"
```
(both at the same time)

Reply:
it will match `-Incidents$` so it will work, but it should not happen

Reply:
ah right, $